File: ViewModelsLayer/ViewModels/UserViewModels/ResponseUserViewModel.cs
using System.Collections.Generic;
using ViewModelsLayer.ViewModels.CardViewModels;
using EntitiesLayer.Enums;

namespace ViewModelsLayer.ViewModels.UserViewModels
{
    public class ResponseUserViewModel
    {
        public int Id { get; set; }
        public string Nickname { get; set; }
        public int Cash { get; set; }
        public UserRoleType UserRole { get; set; }
        public List<ResponseCardViewModel> Cards { get; set; }
    }
}

File: API/Angular/src/app/viewmodels/UserViewModels/response-user-view-model.ts
import { UserRole } from '../../shared/enums/user-role'
import { ResponseCardViewModel } from '../CardViewModels/response-card-view-model';

export class ResponseUserViewModel
{
  constructor(id, nickname, userRole, cards, cash)
  {
    this.id = id;
    this.nickname = nickname;
    this.userRole = userRole;
    this.cards = cards;
    this.cash = cash;
  }

  public id: number;
  public nickname: string;
  public cash: number;
  public userRole: UserRole;
  public cards: Array<ResponseCardViewModel>;
}
